### ISSUE TITLE
Module Request: gate sequencer

### ISSUE THREAD

**Issue by emmahasapetgiraffe** [zezic/ZZC#35]
the left side of phaseque is something ive been missing in other sequencers. i would love to see this in a 16 step sequencer, say 8 lines, maybe things like length and/or probability.

**Comment by zezic** [comment]
Thanks for request. We need to think about how to make it not too similar to what we already have in VCV Pulse Matrix.
When saying about the left part of Phaseque you meant its pattern switching functionality or something else?
Length and probability are good ideas, not sure yet how to fit that many controls in 16x8 grid though.

**Comment by emmahasapetgiraffe** [comment]
yes, the pattern/sequence switching but also the mutating and shifting, the latter being a nice tool to experiment rhythmically since it allows for less rigid rhythms. i notice for me the overall 'feel' of this phase driven vs the typical pulse clocking is less rigid, its allows time to 'wobble' slightly. that is something very musical to me and is something id love to see geared towards rhythm specifically.
I've been thinking about your remark on how to fit so many controls onto a module. maybe it would be possible to 'link' patterns/sequences together in a way of for example have a kick on pattern 1, and a snare on pattern 2, a hihat on pattern 3, have each pattern have their own output, e.g. you connect the kick to output 1, the snare to output 2, hihat to output3. switching between them to program would be selecting a pattern like you would now on the phaseque, but the 'linked' patterns would play all at the same time.
I hope its clear what im trying to say :) please dont hesitate to say if im fuzzy, then i'll try again

**Comment by abozzelli** [comment]
+1 for a gate sequencer with pattern sequencing, resolution and shift (or whatever).

**Comment by mhetrick** [comment]
Phaseque is one of my favorite Rack purchases, but it's better for sequencing a single voice instead of, say, a rack of 8 different drums. What I would love to see is a very simple 8/16 step gate sequencer driven by PHASE. That way, we can use the excellent Clock and Divider modules to create bendable phases. The Pulse Matrix is great, but it is clocked only by triggers. The ability to manipulate the phase would give sequencing options similar to the ESQ by K-Devices (https://k-devices.com/products/esq/).
